tests/test-cli-coverage-gaps.sh

```
@@ -0,0 +1,105 @@
+chaos_output=$(zipper --chaos-list 2>&1) || true
+if echo "$chaos_output" | grep -q "full-chaos"; then
+    pass "--chaos-list contains scenario names"
+else
+    fail "--chaos-list contains scenario names"
+fi
+
+# --- Multi-format generation ---
+
+print_info "Test: --load-file-formats dat,opt,csv"
+zipper --type pdf --count 5 --output-path "$TEST_OUTPUT_DIR/multi_format" --load-file-formats dat,opt,csv > /dev/null 2>&1
+[[ -n "$(find "$TEST_OUTPUT_DIR/multi_format" -name "*.dat" -print -quit)" ]] && pass "produces .dat" || fail "produces .dat"
```

Shellcheck (0.11.0)

[info] Note that A && B || C is not if-then-else. C may run when A is true.

(SC2015)